### PR TITLE
Assessment tool landing page version 1

### PR DESCRIPTION
**Pull request by ElminaIusifova**

### CODE REVIEW

File: _pages/en/capabilities.md [hunk 1]
```diff
@@ -71,7 +30,66 @@ Average of "How much time" questions
...
+
+#### What is the problem and solution  
+
+To manage today business environment challenges organizations need the ability to deliver software with speed and stability. It will help to improve technology and organizational performance.  
+
+#### Who is it for  
+
+IT teams in IITB but any Dev team can benefit from this tool.
```
Comment (gcharest) [inline _pages/en/capabilities.md:43]: I'm thinking this sentence should not specifically identify devs or at least expand beyond the devs?

Would **"All teams in IITB looking to continuously improve themselves and support the improvement efforts of their peers"** be somewhat accurate?

File: _pages/en/capabilities.md [hunk 2]
```diff
@@ -71,7 +30,66 @@ Average of "How much time" questions
...
+For better outcomes use this path for continuous integration and delivery:  
+![A graphic with 4 circles demonstrating assessment cycle process. There are 4 circles connected with an arrow  in clockwise direction.
+The circle titles are: top "Assess"; right "Analyze"; bottom "Implement"; left "Improve".](assets/images/assessment_process_improvement.png){:height="100%" width="100%" style="display:block;margin-left:auto;margin-right:auto"}  
+
+#### Assessment tool process  
+
+Our process includes:  
+
+- Invitation to take a survey  
+- Choose a section to be assessed
```
Comment (gcharest) [inline _pages/en/capabilities.md:57]: Could we say something like: **Choose a topic to self-assess** instead?

**Review by gcharest — CHANGES_REQUESTED** [review]
Overall comment: really good! Seems to introduce a positive narrative to the whole process!
A small change request:
- Try to put one sentence per line (I'll try to make those changes) so that changes to the whole file is more granular. The reason is that Git tracks line level changes; when we put many sentences on a single line it makes harder to spot the small differences.

**Comment by gcharest** [comment]
Just wondering if this closes #1367 ?

File: _pages/en/capabilities.md (hunk 1 above)
Comment (ElminaIusifova) [reply _pages/en/capabilities.md]: Changed! I like it better ;) 
Thanks for letting me know about one sentence per line, I didn't know about it;  **"learning git never stops"**

File: _pages/en/capabilities.md (hunk 2 above)
Comment (ElminaIusifova) [reply _pages/en/capabilities.md:89]: yap! Changed as well! thanks G

**Comment by smellems** [comment]
This is good for the capabilities page and maybe we can reuse for the landing page.  But I think the landing page #1367, should be a separate page from the full list of questions.

With a link to the live survey - https://sara-sabr.github.io/dora-js/

**Review by smellems — APPROVED** [review]

**Comment by ElminaIusifova** [comment]
Eventually, we will take the questions off the website and leave just the link to our tool on landing page

**Review by gcharest — APPROVED** [review]

**Pull request merged**